internal/server/v3/resolve/golden/resolve_by_id.json

```diff
@@ -0,0 +1,15 @@
+{
+  "entities": [
+    {
+      "node": "undata-geo:G00003340",
+      "candidates": [
+        {
+          "dcid": "Q30"
```

Comment:
This matches the behavior in V2, but maybe we can think about whether we should update the response fields, since this isn't really a "dcid"